### PR TITLE
Minor clarifications and consistency

### PR DESCRIPTION
**Pull request by drewnoakes**
A few small edits that I feel make things a bit clearer or more specific.

### CODE REVIEW

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ In order to get around this error, you can either remove the explicit `Compile`
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
 </PropertyGroup>
 ```
-Setting this property to `false` will override implicit inclusion and the behavior will revert back to the previous SDKs where you had to specify the default globs in your project. 
+Setting this property to `false` will disable implicit inclusion and the behavior will revert back to that of the previous SDKs where you had to specify the default globs in your project.
```
Comment (jzeferino) [inline docs/core/tools/csproj.md]: "to that of" sounds weird.

**Review by rpetrusha — APPROVED** [review]
Thanks for adding these clarifications, @drewnoakes. I'll merge when the build completes.

File: docs/core/tools/csproj.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ If the project has multiple target frameworks, the results of the command should
 ## Additions
 
 ### Sdk attribute 
-The `<Project>` element of the *.csproj* file has a new attribute called `Sdk`. `Sdk` specifies which SDK will be used by the project. The SDK, as the [layering document](cli-msbuild-architecture.md) describes, is a set of MSBuild [tasks](/visualstudio/msbuild/msbuild-tasks) and [targets](/visualstudio/msbuild/msbuild-targets) that can build .NET Core code. We ship three main SDKs with the .NET Core tools:
+The root `<Project>` element of the *.csproj* file has a new attribute called `Sdk`. `Sdk` specifies which SDK will be used by the project. The SDK, as the [layering document](cli-msbuild-architecture.md) describes, is a set of MSBuild [tasks](/visualstudio/msbuild/msbuild-tasks) and [targets](/visualstudio/msbuild/msbuild-targets) that can build .NET Core code. We ship three main SDKs with the .NET Core tools:
```
Comment (jzeferino) [inline docs/core/tools/csproj.md]: LGTM.

File: docs/core/tools/csproj.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ Alternatively, the attribute can contain:
 * `All` – all assets are used.
 
 ### DotNetCliToolReference
-`<DotNetCliToolReference>` item element specifies the CLI tool that the user wants to restore in the context of the project. It's 
+A `<DotNetCliToolReference>` item element specifies the CLI tool that the user wants to restore in the context of the project. It's
```
Comment (jzeferino) [inline docs/core/tools/csproj.md]: LGTM.

**Review by jzeferino — CHANGES_REQUESTED** [review]

File: docs/core/tools/csproj.md (hunk 1 above)
Comment (rpetrusha) [reply docs/core/tools/csproj.md]: it's actually more correct than the original. It's the behavior that reverts, not the SDK.

**Comment by rpetrusha** [comment]
Closing and reopening to begin a new build; the previous build failed with a unidentifiable error message.

File: docs/core/tools/csproj.md (hunk 1 above)
Comment (drewnoakes) [reply docs/core/tools/csproj.md]: Would this be clearer?

> Setting this property to `false` will disable implicit inclusion, restoring the behavior of previous SDKs where you had to specify the default globs in your project.

Comment (jzeferino) [reply docs/core/tools/csproj.md]: yes!

Comment (drewnoakes) [reply docs/core/tools/csproj.md]: Done.

**Pull request merged**